packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -525,6 +526,41 @@ abstract class AnimatedWidgetBaseState<T extends ImplicitlyAnimatedWidget> exten
 /// of [Curves.fastOutSlowIn].
 /// {@animation 250 266 https://flutter.github.io/assets-for-api-docs/assets/widgets/animated_container.mp4}
 ///
+/// {@tool snippet --template=stateful_widget_scaffold}
+///
+/// The following example transitions an AnimatedContainer between two states.
```

Comment:
Maybe mention that the transition is triggered by tapping (since you added a gesture detector to the example).

Reply:
👍 fixed